test/e2e_node/swap_test.go

```diff
@@ -103,7 +106,9 @@ var _ = SIGDescribe("Swap", "[LinuxOnly]", nodefeature.Swap, func() {
 				pod := getSwapTestPod(f, qosClass, memoryRequestEqualLimit)
 				pod = runPodAndWaitUntilScheduled(f, pod)
 
-				gomega.Expect(isPodCgroupV2(f, pod)).To(gomega.BeTrueBecause("cgroup v2 is required for swap"))
+				if !isPodCgroupV2(f, pod) {
+					e2eskipper.Skipf("swap tests require cgroup v2")
+				}
```

Comment:
Should we also skip under the `f.Context(framework.WithSerial()` context?

As opposed to the node conformance test, this will only run as part of the swap-conformance lanes, and perhaps it should be very visible when they fail instead of silently skipping? WDYT?

Reply:
ah yeah good point. I'll fix that and add a note